.circleci/config.yml

```diff
@@ -63,6 +61,62 @@ run_e2e_and_save_artifacts: &run_e2e_and_save_artifacts
         path: /tmp/repo/end-to-end-test/shared/imageCompare.html
         destination: /imageCompare.html
 
+
+run_IE11_e2e_and_save_artifacts: &run_IE11_e2e_and_save_artifacts
```

Comment:
this is hugely repetitive of the original e2e task.  i tried to parametize it but at the end of the day just wasn't smart enough to do it.

Reply:
Maybe good to make the `Spin up frontend over ssl if necessary and run end to end tests` a bash script so at least that part has no duplication? Not essential but would be nice